pages/index.js

```diff
@@ -12,6 +13,7 @@ export default function HomePage({ homePageData }) {
 
   return (
     <>
+      <Navigation navigationData={navigationData} />
```

Comment:
I assume you want this to show up on every page – you can either put it in _app.js or [document.js](https://nextjs.org/docs/advanced-features/custom-document). If not, this is fine!

Reply:
Based on the documentation seems like it _app.js and _document.js don't support Data Fetching methods like getStaticProps or getServerSideProps.